src/components/Login.vue

```diff
@@ -1,42 +1,66 @@
 <template>
-  <div class="container">
-    <div class="col m6">
-      <h2 class="center-align">
-        <img src="../assets/logo.png" alt="Welcome to the Kuzzle Backoffice" />
-      </h2>
+  <div class="login">
+    <div class="container">
       <div class="row">
-        <form class="col s6 offset-s3" id="loginForm" method="post" @submit.prevent="login(username, password)">
+        <div class="col card wrapper s10 offset-s1 m8 offset-m2 l4 offset-l4">
           <div class="row">
-            <div class="input-field col s12">
-              <input id="username" v-model="username" type="text" name="username" required
-                     class="validate"/>
-              <label for="username">Email</label>
+            <div class="col s12">
+              <h2 class="center-align logo">
+                <img src="../assets/logo.png" alt="Welcome to the Kuzzle Backoffice" style="width: 70%" />
+              </h2>
             </div>
           </div>
           <div class="row">
-            <div class="input-field col s12">
-              <input v-model="password" type="password" name="password" id="pass" required
-                     class="validate"/>
-              <label for="pass">Password</label>
-            </div>
-          </div>
-          <div class="row">
-            <div class="col m6">
-              <p class="error">{{error}}</p>
-            </div>
-            <div class="col m6">
-              <p class="right-align">
-                <button class="btn waves-effect waves-light" type="submit" name="action">Login</button>
-              </p>
-            </div>
+            <form class="col s10 offset-s1" id="loginForm" method="post" @submit.prevent="login(username, password)">
```

Comment:
Why do we send `username` and `password` in parameters? We have directly access in method with `this.username` thanks to `v-model`